examples/read_ide_file.py

```diff
@@ -34,7 +34,7 @@
 # Loading an ide file is very simple, the supporting files are defined by
 # default in idelib.  This function returns a Document object which contains
 # the data for the given file
-doc = idelib.importFile('../test.ide')
+doc = idelib.importFile('../test.ide', updater=idelib.importer.TQDMUpdater())
```

Comment:
The comment should probably call out and identify the updater

Reply:
agreed and fixed.

Reply:
@CrepeGoat @SamRagusa either of you guys want to test this in a jupyter notebook/colab to see if it looks good?